### PR TITLE
Update demo docs and AEA packages plug-in dependencies

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Update demo docs and AEA packages plug-in dependencies

- add `aea-crypto-*` to `dependencies` section in `aea-config.yam` according to docs and demo;
- other minor issues

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

File: aea/configurations/base.py [hunk 1]
```diff
@@ -1304,6 +1304,7 @@ def json(self) -> Dict:
                 "logging_config": self.logging_config,
                 "registry_path": self.registry_path,
                 "component_configurations": self.component_configurations_json(),
+                "dependencies": dependencies_to_json(self.dependencies),
```
Comment (marcofavorito) [inline aea/configurations/base.py:1307]: if `dependencies: {}`, then it was removed without these changes

File: tests/test_docs/test_ledger_integration.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def _assert_isinstance(self, locals_key, cls_or_str, locals_):
 
     def _assert(self, locals_, *mocks):
         """Assert code outputs."""
-        self._assert_isinstance("aea_crypto_fetchai", "FetchAICrypto", locals_)
+        self._assert_isinstance("fetchai_crypto", "FetchAICrypto", locals_)
```
Comment (marcofavorito) [inline tests/test_docs/test_ledger_integration.py:67]: it was just a wrong renaming

File: docs/ledger-integration.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ These three classes have their own registries, which allow the developer to impo
 from aea.crypto.registries import crypto_registry, make_crypto, register_crypto
 
 # by default we can use the native cryptos
-aea_crypto_fetchai = make_crypto("fetchai")
+fetchai_crypto = make_crypto("fetchai")
```
Comment (marcofavorito) [inline docs/ledger-integration.md:25]: it was just a wrong renaming

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**